pkg/build/nodeimage/defaults.go

```diff
@@ -20,7 +20,7 @@ package nodeimage
 const DefaultImage = "kindest/node:latest"
 
 // DefaultBaseImage is the default base image used
-const DefaultBaseImage = "kindest/base:v20200423-30be2258"
+const DefaultBaseImage = "aojea/kindbase:auto"
```

Comment:
this is built with current PR, it's being added on a temporal commit just to test it works on CI

Reply:
I'll rebase and push an image after reviewing the other changes 👍